tests/rhui3_tests/test_client_management.py

```diff
@@ -5,6 +5,10 @@
 
 from os.path import basename
 
+# The parts related to Atomic can be skipped because they only work on RHEL 7; BZ 1405083.
```

Comment:
I would replace with "The parts related to Atomic are applicable to RHEL 7 only". Otherwise 'can be skipped' sounds like there is some 'main' testing platform, i.e. RHEL6. And then there is no need to mention that BZ